src/components/FindMovie/FindMovie.tsx

```diff
@@ -1,54 +1,94 @@
-import React from 'react';
+/* eslint-disable no-console */
+import React, { useState } from 'react';
+import { MovieCard } from '../MovieCard';
 import './FindMovie.scss';
 
 // import { MovieCard } from '../MovieCard';
 
-export const FindMovie: React.FC = () => (
-  <>
-    <form className="find-movie">
-      <div className="field">
-        <label className="label" htmlFor="movie-title">
-          Movie title
-        </label>
-
-        <div className="control">
-          <input
-            type="text"
-            id="movie-title"
-            placeholder="Enter a title to search"
-            className="input is-danger"
-          />
-        </div>
+interface Props {
+  addMovieToList: (movie:Movie | undefined) => void
+}
 
-        <p className="help is-danger">
-          Can&apos;t find a movie with such a title
-        </p>
-      </div>
+export const FindMovie: React.FC<Props> = ({ addMovieToList }) => {
+  const [movie, setMovie] = useState<Movie | undefined>(undefined);
```

Comment:
```suggestion
  const [movie, setMovie] = useState<Movie | null>(null);
```
Используй null в качестве дефолтного значения